ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "bulmaswatch",
+    "version": "0.4.1",
+    "description": "Themes for Bulma",
+    "author": "Jenil Gogari <i@jgog.in>",
+    "homepage": "https://jenil.github.io/bulmaswatch",
+    "filename": "default/bulmaswatch.min.css",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/jenil/bulmaswatch.git"
+    },
+    "license": "MIT",
+    "keywords": [
+        "css",
+        "sass",
+        "flexbox",
+        "responsive",
+        "framework",
+        "themes"
+    ]
+}
```

Comment:
Let's add `autoupdate`, it can be configured to be automatically updated.
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/jenil/bulmaswatch.git",
    "fileMap": [
      {
        "basePath": "default",
        "files": [
          "bulmaswatch*.+(js|css|map)"
        ]
      }
    ]
  }
```
If you have any problemwith auto-update, you could refer to this link --> https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md#2-configure-auto-update